Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.h

```diff
@@ -87,6 +88,14 @@ class GDKTESTGYMS_API ABenchmarkGymGameModeBase : public AGameModeBase
 	bool bExpectedActorCountsInitialised;
 	int32 ActivePlayers; // All authoritative players from all workers
 
+	// For actor migration count
+	int32 PreviousTickMigration;
+	TArray<int32> ToBeRemovedMigration;
```

Comment:
Prefer naming this something like MigrationDeltas, also we should be dividing the Migration count by the number of frames in the array so that we are getting an average per tick of migrating players. From this metric we can represent a minimum percentage of the total number of players as an expectation.